postfix/config.sls

```diff
@@ -20,7 +20,7 @@ include:
     - watch_in:
       - service: postfix
     - template: jinja
-
+{% if salt['pillar.get']('postfix:manage_master_config', True) == True %}
```

Comment:
Change ``if salt['pillar.get']('postfix:manage_master_config', True) == True`` to ``if salt['pillar.get']('postfix:manage_master_config', False)`` since the default behavior should be what was before.